iothub_client/tests/iothubtransport_mqtt_common_ut/iothubtransport_mqtt_common_ut.c

```diff
@@ -150,6 +151,12 @@ static const char* my_Transport_GetOption_Product_Info_Callback(void* ctx)
     return "product_info";
 }
 
+static const char* my_Transport_GetOption_MODEL_ID_Callback(void* ctx)
+{
+    (void)ctx;
+    return "dtmi:testDeviceCapabilityModel;1";
```

Comment:
this has a `;` and the other examples in the above file use a `:`

Intended?
Also ignorance here. Why is this prefixed with `dtmi` and the others `urn`

Reply:
The `urn` is a legacy thing that slipped.  DTDLv1 used urn and DTDLv2 uses dtmi.   *Long* story!!

I'm updating though so we're consistent and on dtmi.